### PR TITLE
[docs] [xcluster] Add min version for semi-automatic mode

### PR DESCRIPTION
**Pull request by hari90**
Min version is 2024.1.2

### CODE REVIEW

**Comment by hari90** [comment]
@netlify /preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel/

**Comment by hari90** [comment]
https://github.com/netlify /preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel/

**Comment by hari90** [comment]
@netlify /preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel/

File: docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ type: docs
   </li>
 </ul>
 
+{{< note title="Note" >}}
+Both primary and standby universes must be on version `2024.1.2` or higher in order to use semi-automatic xCluster mode.
```
Comment (ddhodge) [inline docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md:32]:
```suggestion
To use semi-automatic transactional xCluster replication, both the primary and standby universes must be running v2024.1.2 or later.
```

File: docs/content/stable/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md [hunk 2]
```diff
@@ -28,6 +28,10 @@ type: docs
   </li>
 </ul>
 
+{{< note title="Note" >}}
+Both primary and standby universes must be on version `2024.1.2` or higher in order to use semi-automatic xCluster mode.
```
Comment (ddhodge) [inline docs/content/stable/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md:32]:
```suggestion
To use semi-automatic transactional xCluster replication, both the primary and standby universes must be running v2024.1.2 or later.
```

**Review by ddhodge — APPROVED** [review]

**Pull request merged**